docs/notebooks/getting_started.ipynb

```diff
@@ -0,0 +1,1918 @@
+{
```

Comment:
<p>Not sure whether here the recommended function to use is <code>anndata.read_h5ad</code> or  <code>scanpy.read_h5ad</code>. Maybe   @gtca or @ivirshup can comment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scverse-tutorials/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>talked to @gtca on the train - apparently the Scanpy function will cease to exist soon and move to anndata, so I'd keep the anndata function here</p>